articles/rich-text-inputs.mdx

```diff
@@ -85,17 +74,19 @@ sidebar_html: |
 
 ## Markdown
 
```

Comment:
Somewhere in this section we should link to the new https://cloudcannon.com/documentation/articles/configure-your-markdown-engine/ page

(e.g. near the "SSGs vary in processing Markdown, so using the value from this input depends on your specific implementation." callout, but shown for every ssg)